### ISSUE TITLE
Iran nuclear deal

### ISSUE THREAD

**Issue by stephenbi** [remindmodel/remind#1272]
Capacity factors for tnrs in MEA decline from 75% to 50% utilization from 2005 - 2020. The root cause is very likely the international agreement to limit uranium enrichment in Iran, the only country with nuclear power in MEA.

We currently assume that regions converge to a global default utilization rate by 2035, which is 80% for tnrs. The next revision of input data (6.337) is likely to reflect these numbers, but they may not be realistic.

It's unclear whether this reflects lower output across all nuclear plants in general or if some capacities were turned off. @Renato-Rodrigues argues that it is notoriously difficult to halt and restart nuclear plants, so any capacities turned off could probably be considered retired. (Thus, 80% utilization of these 'existing' capacities by 2035 may be unrealistic).

On the other hand, Iran hasn't fully complied with the agreement, which the US pulled out of, and there is already a new 300 MW plant under construction set to come online by 2030.

**Comment by Renato-Rodrigues** [comment]
I would rephrase the sentence assigned to me, from "should be considered retired" to "could probably be considered retired". As this demands some more research to see the real status of the inactive nuclear power plants.

**Comment by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q** [comment]
This makes no sense in a whole lot of dimensions.  What data are we talking about?

**Comment by stephenbi** [comment]
Energy Balances generation data and Ember capacity data

**Comment by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q** [comment]
Iran has one reactor with 915 MW capacity that started operation in September 2011, reached full capacity in August 2012 and started commercial operation only in September 2013.
Maybe set capacity factors to zero if capacity is zero, instead of letting them float.
